book/src/road-map.md

```diff
@@ -0,0 +1,33 @@
+### Vendor Management
+
+- [ ] Manual SBOM upload screen (under development)
+- [ ] Role-based permissions related to SBOM upload screen (under development)
+- [ ] Vendor SBOM staleness/freshness policy tracking and enforcement (research & development)
+- [ ] Machine to machine vendor upload mechanism (research)
+
+### SBOM generation & synchronization
+- [X] Snyk
+- [X] GitHub Action (POC)
+- [ ] GitHub (in progress)
+- [ ] GitLab
+- [ ] Bitbucket
+- [ ] Syft
+- [ ] Ion Channel
+
+### SBOM storage
+- [X] S3
+- [X] File system
+
+### Enrichment
+- [X] Snyk
+- [X] EPSS
+- [ ] Ion Channel
+- [ ] OSV
```

Comment:
We need to do the research work that tells us that there is something valuable in OSV that's not already in Snyk.  The whole vulnerability differential conversation.

Reply:
Or that Snyk has something OSV doesn't even. I'm really feeling handwavy about this Roadmap TBH. It was mostly as a conversation starter. My main goal with the PR is to get the documentation started and then refine it over time. We could put something like a `(research)` parenthetical if we think it's important to make a distinction between research and commitments.